### PR TITLE
handle svg background images correctly

### PR DESCRIPTION
**Pull request by janj**
When loading SVG file for the background image we need to first group all the components so when we try to resize we resize the entire group.

https://groups.google.com/g/fabricjs/c/cSPURIBwSL4
https://kiddom.atlassian.net/browse/I20-129

### CODE REVIEW

File: src/SketchField.jsx [hunk 1]
```diff
@@ -601,16 +601,24 @@ class SketchField extends PureComponent {
...
   setBackgroundFromDataUrl = (dataUrl, options = {}) => {
     let canvas = this._fc;
-    let img = new Image();
-    img.setAttribute('crossOrigin', 'anonymous');
+
     const { stretched, stretchedX, stretchedY, ...fabricOptions } = options
-    img.onload = () => {
-      const imgObj = new fabric.Image(img);
+    let imgLoadFunc = fabric.Image.fromURL
+    if (dataUrl.endsWith('.svg') || dataUrl.includes('.svg?')) {
```
Comment (janj) [inline src/SketchField.jsx:607]: maybe there's a better way to indicate svg?

Comment (nguyen-dunn) [reply src/SketchField.jsx:607]: @janj 
Does this work for you? I'd had some issues in when trying to load mathml generated svg into canvas. If we make this change here we would need to 
- publish this update
- update reference in web canvas sketch is already moved from kiddom components

Comment (janj) [reply src/SketchField.jsx:607]: yeah this is working for me locally
I'm not sure what the rest of your comment means though, sorry

**Review by mikedoty — APPROVED** [review]
Thx for doing this!

**Pull request merged**